### ISSUE TITLE
Doesn't work in submodules

### ISSUE THREAD

**Issue by expipiplus1** [anchor/git-vogue#92]
It should insert the hooks in the directory specified in `.git`.

**Comment by ctd** [comment]
Hi, thanks for the bug report.

Unfortunately, this code isn't being actively maintained anymore by Anchor. As such, it's unlikely this will be fixed.

**Comment by olorin** [comment]
> Unfortunately, this code isn't being actively maintained anymore by Anchor. As such, it's unlikely this will be fixed.

See also #91; this repository may have a new home at some stage at which point others may be able to take a more active role in maintainership. /cc @oswynb

**Comment by expipiplus1** [comment]
Yeah, at the moment its not possible to raise issues in that fork. Here seems like the next best place.

**Comment by christian-marie** [comment]
Ready to accept patches now, shoot!

**Comment by christian-marie** [comment]
Is this still something you use? I wonder if simply 'git-vogue init'-ing in the submodules would achieve your goals. Please re-open if you still want this and I'll make a cleaner issue with a path forward. Doesn't sound too tricky.

**Comment by expipiplus1** [comment]
Hi, sorry I'm no longer in need of this!
